<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Disparity</class>
 <widget class="QDialog" name="Disparity">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1329</width>
    <height>587</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Visually measure the minimum and maximum disparity</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>550</y>
     <width>621</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_disparity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>650</width>
     <height>471</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the slider until the BACKGROUND is sharp</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_foreground_disparity">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>650</width>
     <height>471</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the slider until the FOREGROUND is sharp</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_background_disparity">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>650</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the slider until the BACKGROUND is sharp</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="horizontalSlider_foreground_disparity">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>510</y>
     <width>650</width>
     <height>29</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the slider until the FOREGROUND is sharp</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_background_disparity_value">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>530</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the slider until the BACKGROUND is sharp</string>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_foreground_disparity_value">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>530</y>
     <width>63</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Use the slider until the FOREGROUND is sharp</string>
   </property>
   <property name="styleSheet">
    <string notr="true">text-align:center;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>284</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>BACKGROUND</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>20</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="text">
    <string>FOREGROUND</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>5</y>
     <width>351</width>
     <height>31</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>Visually measure the minimum and maximum disparity</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Disparity</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Disparity</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_background_disparity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_background_disparity_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>334</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>341</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_foreground_disparity</sender>
   <signal>valueChanged(int)</signal>
   <receiver>label_foreground_disparity_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>994</x>
     <y>494</y>
    </hint>
    <hint type="destinationlabel">
     <x>991</x>
     <y>507</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
